machine_learning/local_weighted_learning/local_weighted_learning.py

```diff
@@ -0,0 +1,133 @@
+# Required imports to run this file
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# weighted matrix
+def weighted_matrix(point: np.mat, training_data_x: np.mat, bandwidth: float) -> np.mat:
+    """
+    Calculate the weight for every point in the
+    data set. It takes training_point , query_point, and tau
+    Here Tau is not a fixed value it can be varied depends on output.
+    tau --> bandwidth
+    xmat -->Training data
+    point --> the x where we want to make predictions
+    >>> weighted_matrix(np.array([1., 1.]),np.mat([[16.99, 10.34], [21.01,23.68],
+                        [24.59,25.69]]), 0.6)
```

Comment:
You just need to use the `...` to show doctest that we are continuing the previous line.
```suggestion
    >>> weighted_matrix(np.array([1., 1.]),np.mat([[16.99, 10.34], [21.01,23.68],
    ...                 [24.59,25.69]]), 0.6)
```